test/e2e-addon-test/addon_test.go

```diff
@@ -121,4 +121,15 @@ var _ = Describe("Addon tests", func() {
 			},
 			time.Second*30, time.Millisecond*500).ShouldNot(BeNil())
 	})
+
+	PIt("Addon observability is successfully enabled", func() {
```

Comment:
Why PIT this?

Reply:
Addon observability is not allowed to be integrated to `oam-dev/catalog/addons` until this PR passed. So I have to escape this test and enable it later.